DTS-STN/Life-Journeys
# Update Next & Tailwind dependencies

# Description

App was broken due to Tailwind and Next being outdated, this PR updates Tailwind to v3, Next to latest and removes the unnecessary `fonts.css` file and instead imports the fonts in `globals.css`

## Test Instructions

1. Pull in branch
2. Type `npm run dev`
3. Confirm app runs without errors
